src/node_contextify.cc

```diff
@@ -184,6 +195,34 @@ Local<Context> ContextifyContext::CreateV8Context(
   CHECK(name->IsString());
   Utf8Value name_val(env->isolate(), name);
 
+  Local<Value> codegen = options_obj->Get(env->context(),
+      FIXED_ONE_BYTE_STRING(env->isolate(), "codeGeneration"))
+          .ToLocalChecked();
+
+  if (!codegen->IsUndefined()) {
+    CHECK(codegen->IsObject());
+    Local<Object> codegen_obj = codegen.As<Object>();
+
+    Local<Value> allow_code_generation_from_strings =
+      codegen_obj->Get(env->context(),
+          FIXED_ONE_BYTE_STRING(env->isolate(), "strings"))
+      .ToLocalChecked();
+    if (!allow_code_generation_from_strings->IsUndefined()) {
+      CHECK(allow_code_generation_from_strings->IsBoolean());
+      ctx->AllowCodeGenerationFromStrings(
+          allow_code_generation_from_strings.As<Boolean>()->Value());
+    }
+
+    Local<Value> allow_wasm_code_generation = codegen_obj->Get(env->context(),
+        FIXED_ONE_BYTE_STRING(env->isolate(), "wasm"))
+      .ToLocalChecked();
+    if (!allow_wasm_code_generation->IsUndefined()) {
+      CHECK(allow_wasm_code_generation->IsBoolean());
+      ctx->SetEmbedderData(kAllowWasmCodeGeneration,
```

Comment:
Better set it unconditionally because calling GetEmbedderData() on an undefined index is undefined behavior.

Reply:
A better idea seems to be setting the slot to false unconditionally in `NewContext()` and then override the value in contextity, since the wasm callback is called on the main context as well.

Reply:
i'll do tgu's approach

sidenote @nodejs/v8 is there any reason why we don't have `Context::AllowWasmCodeGeneration`/can it be added?

Reply:
could the current approach here cause issues for people making native modules and creating a new context under node's isolate?